x-pack/legacy/plugins/index_management/public/components/mappings_editor/constants/field_options_i18n.ts

```diff
@@ -0,0 +1,495 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { i18n } from '@kbn/i18n';
+
+interface Optioni18n {
+  title: string;
+  description: string;
+}
+
+type IndexOptions =
+  | 'indexOptions.docs'
+  | 'indexOptions.freqs'
+  | 'indexOptions.positions'
+  | 'indexOptions.offsets';
+
+type AnalyzerOptions =
+  | 'analyzer.indexDefault'
+  | 'analyzer.standard'
+  | 'analyzer.simple'
+  | 'analyzer.whitespace'
+  | 'analyzer.stop'
+  | 'analyzer.keyword'
+  | 'analyzer.pattern'
+  | 'analyzer.fingerprint'
+  | 'analyzer.language';
+
+type SimilarityOptions = 'similarity.bm25' | 'similarity.boolean';
+
+type TermVectorOptions =
+  | 'termVector.no'
+  | 'termVector.yes'
+  | 'termVector.withPositions'
+  | 'termVector.withOffsets'
+  | 'termVector.withPositionsOffsets'
+  | 'termVector.withPositionsPayloads'
+  | 'termVector.withPositionsOffsetsPayloads';
+
+type OrientationOptions = 'orientation.counterclockwise' | 'orientation.clockwise';
+
+type LanguageAnalyzerOption =
+  | 'arabic'
+  | 'armenian'
+  | 'basque'
+  | 'bengali'
+  | 'brazilian'
+  | 'bulgarian'
+  | 'catalan'
+  | 'cjk'
+  | 'czech'
+  | 'danish'
+  | 'dutch'
+  | 'english'
+  | 'finnish'
+  | 'french'
+  | 'galician'
+  | 'german'
+  | 'greek'
+  | 'hindi'
+  | 'hungarian'
+  | 'indonesian'
+  | 'irish'
+  | 'italian'
+  | 'latvian'
+  | 'lithuanian'
+  | 'norwegian'
+  | 'persian'
+  | 'portuguese'
+  | 'romanian'
+  | 'russian'
+  | 'sorani'
+  | 'spanish'
+  | 'swedish'
+  | 'turkish'
+  | 'thai';
+
+export type FieldOption =
+  | IndexOptions
+  | AnalyzerOptions
+  | SimilarityOptions
+  | TermVectorOptions
+  | OrientationOptions;
+
+export const FIELD_OPTIONS_TEXTS: { [key in FieldOption]: Optioni18n } = {
```

Comment:
It looks like removing `PARAMETERS_OPTIONS` lets us remove `getOptionTexts` which will also allow us to remove `FIELD_OPTIONS_TEXTS`.